src/commands/initProjectForVSCode/InitVSCodeStep/PythonInitVSCodeStep.ts

```diff
@@ -7,29 +7,28 @@ import * as fse from 'fs-extra';
 import * as os from 'os';
 import * as path from 'path';
 import { DebugConfiguration, TaskDefinition } from 'vscode';
-import { extensionPrefix, extInstallCommand, extInstallTaskName, func, funcWatchProblemMatcher, gitignoreFileName, hostStartCommand, packTaskName, Platform, pythonVenvSetting } from "../../../constants";
+import { extensionPrefix, extInstallCommand, extInstallTaskName, func, funcWatchProblemMatcher, gitignoreFileName, hostStartCommand, Platform, pythonVenvSetting } from "../../../constants";
 import { pythonDebugConfig } from '../../../debug/PythonDebugProvider';
 import { venvUtils } from '../../../utils/venvUtils';
 import { IProjectWizardContext } from '../../createNewProject/IProjectWizardContext';
 import { getExistingVenv } from '../../createNewProject/ProjectCreateStep/PythonProjectCreateStep';
 import { ScriptInitVSCodeStep } from './ScriptInitVSCodeStep';
 
 export class PythonInitVSCodeStep extends ScriptInitVSCodeStep {
-    protected preDeployTask: string = packTaskName;
     private _venvName: string | undefined;
 
     protected async executeCore(context: IProjectWizardContext): Promise<void> {
         await super.executeCore(context);
 
-        const zipPath: string = this.setDeploySubpath(context, `${path.basename(context.projectPath)}.zip`);
+        this.settings.push({ key: 'scmDoBuildDuringDeployment', value: true });
```

Comment:
Isn't this always going to be true?  I thought while this was the default, we weren't going to force users to use it.

Reply:
Yes it's always true, but this code only happens when creating a project. After the project is created, they can do whatever they want with their settings.